HISTORY.rst

```diff
@@ -1,6 +1,18 @@
 Release History
 ---------------
 
+master
+++++++
+
+**Features and Improvements**
+
+* Behave's short form arguments are now accepted (e.g. :code:`-i` for :code:`--include`)
+* Added :code:`--keepdb` short form argument, `-k`
+
+**Bugfixes**
+* Fixed specifying paths didn't work
```

Comment:
There is a blank link missing after "Bugfixes", otherwise the bulleted list is not interpreted correctly.

Also, please use `-` signs instead of `*` for bullets, to keep the markup uniform across the document.  :+1: